app/controllers/api/v1/onboarding_follow_recommendations_controller.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+# Not to be confused with [Api::V1::Accounts::FollowRecommendationsController].
+# This API suggests a hardcoded list of follows for new users while they are going through the
+# onboarding flow after signing up for a new account.
+# FollowRecommendationsController is used, instead, to suggest new follows to existing users based on
```

Comment:
I think you're missing a bit here - shouldn't it read "If the user is already following people, FollowRecommendationsController is used, instead..."?

Reply:
I think these 2 APIs are used in different contexts in different parts of the app. You could call `FollowRecommendationsController` even if you're not following anyone, it's just that the suggestions won't be that great hahah

Reply:
ohh, got it